gamemodes/terrortown/gamemode/shared/sh_player_ext.lua

```diff
@@ -852,8 +852,8 @@ end
 -- Returns whether a @{Player} was revived after beeing confirmed this round
 -- @return boolean
 -- @realm shared
-function plymeta:Revived()
-	return not self:TTT2NETGetBool("body_found", false) and self:OnceFound()
+function plymeta:WasRevivedAndConfirmed()
+	return self:WasRevivedInRound() and self:IsTerror() and self:OnceFound() and self:RoleKnown()
```

Comment:
Same as before, the `self:IsTerror` doesn't match the function name, even a current spectator could be revived and confirmed